index.html

```diff
@@ -669,6 +670,12 @@ <h3>
                 <p>
                     The <dfn>pageInputQuery</dfn> attribute contains inputted query for the MiniApp page.
                 </p>
+                
+                <aside class="note">
+                    <p>
+                        By default, the inputted query is the URL parameter's query set by navigate or redirect API by another MiniApp page (for example, "path?key1=value&key2=value2"), so that the MiniApp page can have further implementation based on the inputted query while being loaded.
```

Comment:
```suggestion
                        By default, the inputted query is the URL parameter's query set by navigate or redirect API by another MiniApp page (for example, `path?key1=value&key2=value2`), so that the MiniApp page can have further implementation based on the inputted query while being loaded.
```